Dockerfile

```
@@ -1,6 +1,6 @@
-ARG VERSION=unspecified
+ARG VERSION=1.3.0
```

The `VERSION` argument should be left unspecified- it is meant to be passed in when the image is created, as noted in the [README](https://github.com/cisagov/guacscanner-docker/blob/develop/Dockerfile#L1).  You can also see how the version is [determined](https://github.com/cisagov/admiral-docker/blob/develop/.github/workflows/build.yml#L182-L219) and [used](https://github.com/cisagov/admiral-docker/blob/develop/.github/workflows/build.yml#L253-L254) in the build workflow.
```suggestion
ARG VERSION=unspecified
```